src/NLog.Targets.MauiLog/MauiLogTarget.Android.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Android.OS;
+
+namespace NLog.Targets;
+    
+/// <summary>
+/// Output target for Android.Util.Log
+/// </summary>
+[Target("MauiLog")]
+[NLog.Targets.MauiLogLinker.Preserve]
+public class MauiLogTarget : TargetWithLayoutHeaderAndFooter
```

Comment:
Possible that the new namespace-behavior can be reverted, so it will be git-rename, instead git-delete and git-rename.